docker/quorum-crux/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM alpine:3.8
```

Comment:
This is the Dockerfile to build the full image right? Is none of this stuff going in the Quorum examples repo? We can discuss tomorrow.